views/setlists/view.php

```diff
@@ -59,8 +59,14 @@
     </div>
   </div>
 
-  <a href="<?= $base_url ?>/songs" data-role="button" data-theme="b" <?php if (count($songs) == 0) echo 'disabled' ?>>View All</a>
+  <a href="<?= $base_url ?>/songs" data-role="button" data-theme="b" <?php if (count($songs) == 0) echo 'disabled' ?>>View Chords</a>
+<?php
+  # Not sure if this is best place to do this
```

Comment:
So I had a question about this. I need to generate access_key for old songsets at least before they're viewed. However, is it better to generate access_key for ALL old songsets in migration? or in a maintenance script?